packages/server-wallet/src/engine/engine.ts

```diff
@@ -1027,14 +993,25 @@ export class SingleThreadedEngine
     response.succeededObjectives.map(o => this.emit('objectiveSucceeded', o));
   }
 
+  /**
+   * Update the engine's knowledge about the funding for a channel.
```

Comment:
This method applies to directly funded channels. Do we want to clarify that in the comment?

Reply:
82b36e9524113bbc7094ec46da5477a34065e71f

As the implementation is agnostic to the `fundingStrategy` of the channel, I opted to avoid saying that it applies to directly funded channels.

(`holdingUpdated` may be called for a virtually-funded channel, for instance)